bin/cylc-run

```diff
@@ -95,7 +96,11 @@ warm start unless you use special cold-start tasks. See "Suite Start-up" and
             start_point_string = self.args[1]
             if start_point_string == "now":
                 start_point_string = (
-                    datetime.datetime.utcnow().strftime("%Y%m%dT%H%MZ")
+                    datetime.datetime.now().strftime("%Y%m%dT%H%M")
```

Comment:
Minute level information is likely to be a pain, I think, if the suite has any `[[[PXX]]]` sections.

How about a syntax like this:
 * `now` => `%Y%m%dT%H`
 * `now("%Y%m%dT%H%M")` => `%Y%m%dT%H%M`